### PR TITLE
Upgrade guzzle

### PR DESCRIPTION
**Pull request by addelong**

### CODE REVIEW

File: src/ChartBlocks/Client.php [hunk 1]
```diff
@@ -91,7 +103,8 @@ public function __get($name) {
      * 
      * @return string
      */
-    public function getApiUrl() {
+    public function getApiUrl(): string
```
Comment (addelong) [inline src/ChartBlocks/Client.php:106]: There's quite a few changes in this PR that are the result of hitting the "Cleanup code" button in PHPStorm which basically amounts to adding a return type signature to functions. Just a heads up.

File: composer.json [hunk 2]
```diff
@@ -5,10 +5,11 @@
     "license": "MIT",
     "require": {
         "php": ">=5.4",
-        "guzzle/http": ">=3.6.0,<4"
+        "guzzlehttp/guzzle": "~6.3.3",
+        "ext-json": "*"
```
Comment (addelong) [inline composer.json:9]: Even though ext-json is present by default, it is apparently best practice to explicitly list it as a requirement if you use things like `json_decode`.

File: src/ChartBlocks/Client.php [hunk 3]
```diff
@@ -148,86 +163,64 @@ public function getAuthSecret() {
...
-            $signature = $this->getSignature()->fromRequest($request, $secret);
-            $request->setHeader('Authorization', 'Basic ' . base64_encode($token . ':' . $signature));
-        } elseif ($token xor $secret) {
-            throw new \RuntimeException('Both token and secret must be set');
-        }
+        $body = Psr7\Utils::streamFor(fopen($file, 'r'));
+        $response = $this->getHttpClient()->post($path, [
+            'body' => $body,
+            'headers' => [
+                'Content-Type' => $contentType ?: 'application/octet-stream'
```
Comment (TomasRebro) [inline src/ChartBlocks/Client.php:212]: Was this default content type automatically set behind the scenes before? 🤔

**Review by TomasRebro — APPROVED** [review]
LGTM 👍 I just have one question to better understand that specific change.

**Pull request merged**